src/common/hooks/use-publish-hey.ts

```diff
@@ -41,7 +44,10 @@ export function useHandlePublishContent() {
         contractAddress,
         contractName,
         functionName: MESSAGE_FUNCTION,
-        functionArgs: [stringUtf8CV(content)],
+        functionArgs: [
+          stringUtf8CV(content),
+          attachmentUri !== '' ? someCV(stringUtf8CV(attachmentUri)) : noneCV(),
+        ],
         onFinish: () => {
           _onFinish();
           onFinish();
```

Comment:
I don't get this? What is it doing here `onFinish`?

Reply:
I'm passing another onFinish from the component to this hook

Reply:
I see it now in `handleSubmit`, thanks.